### PR TITLE
HTML Report: add elapsed time

### PR DESCRIPTION
**Pull request by Slamdunk**
Hi, in our business app the coverage takes more than 2 hours, and needs to be planned precisly.

Adding the total elapsed time in the footer can be a great perk for resource management planning.

I don't know if you accept the new dependency requirement of the [PHP_Timer](https://github.com/sebastianbergmann/php-timer), but it's already installed via [PHPUnit\composer.json#30](https://github.com/sebastianbergmann/phpunit/blob/5.1.3/composer.json#L30)

### CODE REVIEW

**Comment by Slamdunk** [comment]
@sebastianbergmann may I ping you for a feedback?

File: composer.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "phpunit/php-file-iterator": "~1.3",
         "phpunit/php-token-stream": "~1.3",
         "phpunit/php-text-template": "~1.2",
+        "phpunit/php-timer": ">=1.0.6",
```
Comment (GrahamCampbell) [inline composer.json]: Bad version constraint because it has no upper bound.

Comment (Slamdunk) [inline composer.json]: I copied it from current [PHPUnit\composer.json](https://github.com/sebastianbergmann/phpunit/blob/5.1/composer.json#L30) from current stable PHPUnit branch, to mantain ubiquity.

What should it be?

**Comment by sebastianbergmann** [comment]
Thank you for your contribution but I do not want this, sorry.

**Pull request closed without merging**